### DISCUSSION TITLE
More Feature Supports in the Future?

### DISCUSSION THREAD

**Discussion by Tolga1452** [copilot-extensions/preview-sdk.js#110] [Q&A] [answered]
Hello! I would like to move [my comment](https://github.com/orgs/community/discussions/137975#discussioncomment-10717083) here since I think it would be better to ask here.

> I have a question (for the future versions of this beta). Will GitHub send references (such as selected code) to the endpoint along with the history, just like how GitHub Copilot works? Because I need to make a VS Code extension to get the code references from the IDE, but if I do that, the extension adds another participant to the chat and probably I need to implement a separate system within the extension, which would differ from the one my server currently uses to handle requests from GitHub, especially because I get different data from the VS Code API.
> 
> I also wonder if other chat extension capabilities will be added. Because I would like to send custom buttons, follow-up messages etc. to the chat. And the most disappointing part is I expected to create commands for my Copilot Extension without having to build an IDE extension.
> 
> I would like to have these features because with the current usage of Copilot Extensions (I also took a look at the [dreamcode](https://github.com/copilot-extensions/preview-sdk.js/blob/main/dreamcode.md)), we can only provide a chat experience to the users, which is far away from helping to code.

**Comment by gr2m** [comment, marked as answer]
I think that https://github.com/orgs/community/discussions/137975 is the best place to ask these questions. With the SDK, we can only implement what the API provides, asking for more API capabilities should be done in https://github.com/orgs/community/discussions

**Reply by Tolga1452** [reply]
Oh okay... I just thought I might be wrong since didn't get a reply for a while (also the thread is quiet). Thank you anyway!

**Reply by gr2m** [reply]
Copilot has its own category in community discussions:
https://github.com/orgs/community/discussions/categories/copilot

I'd suggest to submit different discussions for different feature requests. They are monitored, but there is never a guarantee that folks will respond or when.

In general, most folks at GitHub are focused on the upcoming conference where a lot of announcements will be made: https://githubuniverse.com/, it might explain why activity by Hubbers is lower than usual.

**Reply by Tolga1452** [reply]
Well yeah, I actually just wanted to give a quick feedback under the announcement according to my thoughts. I will consider opening a different discussion for it soon.
